.travis.yml

```diff
@@ -8,6 +8,8 @@ cache:
 # env forces jobs to be created from the top level settings
 env:
 jobs:
+  allow_failures:
+    - script: script/run-integration-test.sh
```

Comment:
I am very surprised this works.
Please use a variable like we do in `coala/coala` repo for the job which is allowed to fail